templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
```

Comment:
I don't think so. We'd have to have that URL instead of the rest of this config.

This works because heroku provides this URL, Rails knows how to parse it, and we just override a single setting (or set it in the first place, in this case).